Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM maven:3.8.1-openjdk-16-slim AS MAVEN_BUILD
```

Comment:
This is not an approved image we should only one image openjdk:16-jdk-oraclelinux7

Reply:
The Dockerfile is a multi-stage build and this is just the maven image use to build and package our application the runtime image is in line 14 of the Dockerfile